src/library/scala/collection/BitSetLike.scala

```diff
@@ -163,6 +207,19 @@ trait BitSetLike[+This <: BitSetLike[This] with SortedSet[Int]] extends SortedSe
     fromBitMaskNoCopy(words)
   }
 
+  /**
+    * Equivalent to Java BitSet.intersects, faster than (set1 & set2).nonEmpty
+    */
+  def intersects(other: BitSet): Boolean = {
+    val len = this.nwords min other.nwords
+    for (i <- 0 until len) {
+      if ((word(i) & other.word(i)) != 0L) {
+        return true
+      }
+    }
+    false
+  }
+
```

Comment:
Our backwards compatibility policy prevents us from adding new API in minor releases. This might be able to make it as a last-minute 2.13.0 addition (although it might already be too late there, too, as RC1 is imminent.)